package.json

```diff
@@ -25,6 +25,7 @@
   },
   "dependencies": {
     "@aws-sdk/client-s3": "^3.499.0",
+    "axios": "^1.7.2",
```

Comment:
we'll also need to move this into the projen file with the other dependancies as this is an automatically generated file :)

Once moved to the projen file you can run `make project` and it'll populate this for ya !